docs/authentication/3id-did/lifecycle.md

```diff
@@ -1,27 +1,28 @@
 # 3ID Lifecycle
 
-This page describes the complete lifecycle of a new 3ID, how the DID document is created, how keys are managed, and how 3ID connect uses these building blocks. Most of the logic described here is implemented in the [3id-did-provider](https://github.com/ceramicstudio/js-3id-did-provider) package.
+This page describes the complete lifecycle of a [3ID DID](./method.md), including how the DID document is created, how keys are managed, and how [3ID Connect](./3id-connect.md) uses these building blocks. Most of the logic described here is implemented in the [3id-did-provider](./provider.md) package.
 
-The two core components to the 3ID lifecycle is the [3ID DID method](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-79/CIP-79.md) and the [3ID Keychain](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-20/CIP-20.md) which is an IDX definition for storing encrypted key material.
+The two core components to the 3ID lifecycle are the [3ID DID method](./method.md) and the [3ID Keychain](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-20/CIP-20.md) which is a standard for storing encrypted key material in a DID's [IDX](../../tools/identity/idx.md).
 
 ## 3ID DID provider lifecycle
+Every 3ID has a *seed* which is the secret used to update the 3ID, sign messages, and decrypt messages as the 3ID. The 3ID DID provider manages this *seed* using *authSecrets*. Each `authSecret` can be used to authenticate to the 3ID and thus get access to the *seed*.
 
-![asdf](../../images/3id-lifecycle.png)
+![3ID DID Provider Lifecycle Diagram](../../images/3id-lifecycle.png)
 
 ### Creating a 3ID
 To create a 3ID the *3id-did-provider* accepts an `authSecret` and an `authId`. It roughly follows the following algorithm.
 
-1. We have an `authSecret A` that is used to create `did:key:A` using the [`key-did-provider-ed25519`](https://github.com/ceramicnetwork/key-did-provider-ed25519) provider
-1. A deterministic TileDocument, referred to as *AuthLink*, is created. If this is a new `authSerect` the loaded *AuthLink* will be empty
-1. A seed for the 3ID is randomly generated, three keys (`controller`, `signing`, and `encryption`) are derived from the seed, and the stream for the 3ID DID document is created
+1. We have an `authSecret A` that is used to create `did:key:A` using the [`key-did-provider-ed25519`](../key-did/provider.md) provider
+1. A deterministic TileDocument, referred to as *AuthLink*, is created. If this is a new `authSecret` the loaded *AuthLink* will be empty
+1. A seed for the 3ID is randomly generated; two keys (`signing`, and `encryption`), and a `did:key:management-1` are derived from the seed. A stream for the 3ID DID document is created with `did:key:management-1` as the *controller*, the `signing` and `encryption` keys are stored in the streams content
```

Comment:
The 'signing' and 'encryption' keys that are put in the stream contents are the *public* keys for signing and encryption of an asymmetric key pair, right?  They can't be symmetric keys or you wouldn't want to put them in the stream contents which is public.  Would be nice to clarify that

Reply:
good point!